templ_enum.cif

```diff
@@ -846,6 +846,10 @@ save_units_code
    'counts'            "counts from a detector"
    'photons per second'  "photons registered in one second"
 
+   'microseconds_per_angstrom' "seconds * 10^(-6)/metres * 10^(-10)"
```

Comment:
I am not familiar with the internal syntax of the description part of the definition so I cannot properly comment on that, however, there definitely seems to be some internal structure to the descriptions (potentially intended to be parsed by dREL?). One thing I noticed, though, is that in other unit definition a negative exponent is used instead of the division sign, see for example 'reciprocal_centimetres'. So maybe the same should also be applied here, i.e.:
```
seconds * 10^(-6)/metres * 10^(-10)
```
could be changed to:
```
( seconds * 10^(-6) ) * ( metres * 10^(-10)^-1 )
```
Also, other descriptions usually contain some additional text components which seemingly refer to the general class of the unit as well as the human-readable name, for example:
```
"length_cubed 'nanometres cubed (metres * 10^( -9))^3'"
```
Maybe something similar should be done here as well?

Reply:
There is no guidance on the appropriate format for these unit explanations, but it makes sense to keep them at least consistent with one another. I will update.